api/client/volume/create.go

```diff
@@ -26,10 +26,17 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [VOLUME_NAME]",
```

Comment:
The convention we use in other commands is `"create [VOLUME]"`, I think it should be the same here.